R/quo.R

```diff
@@ -105,17 +105,16 @@ replace_symbol_in_expr <- function(expression,
 #'
 #' @param order List of expressions
 #'
-#'   *Permitted Values*: list of variables or `desc(<variable>)` function calls
-#'   created by `exprs()`, e.g., `exprs(ADT, desc(AVAL))`
+#' @permitted list of variables or `desc(<variable>)` function calls created by
```

Comment:
```suggestion
#' @permitted list of variables or `dplyr::desc(<variable>)` function calls created by
```
While I am not a prefix person - wondering if this might be helpful for users or even devs is we put this in there??

Reply:
I wouldn't add the prefix because it suggests that it is required. But it isn't. You don't even need the prefix if `dplyr` is not loaded.